src/ontology/hp-edit.owl

```diff
@@ -174872,7 +174872,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033121> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0033122> (Absent P wave)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033122> "The P wave that normally precedes each QRS complex by a fixed PR interval of 120 to 200 milliseconds is not present.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033122> "The P wave that normally precedees each QRS complex by a fixed PR interval of 120 to 200 milliseconds is not present.")
```

Comment:
original was correct

Reply:
should be fixed now :) thanks for catching